ui/admin/app/components/form/credential-store/index.hbs

```diff
@@ -6,12 +6,47 @@
   @showEditToggle={{if @model.isNew false true}}
   as |form|
 >
-  <form.input
-    @value={{@model.type}}
-    @label={{t 'form.type.label'}}
-    readonly={{true}}
-    @disabled={{true}}
-  />
+  {{#if (feature-flag 'static-credentials')}}
+    <form.fieldset>
+      <:title>{{t 'form.type.label'}}</:title>
+      <:body>
+        {{#if @model.isNew}}
+          {{! Code to allow option to select between cred. store types (vault or static) when it is first created goes here}}
```

Comment:
Not sure if this comment was left on purpose.

Reply:
Yeah, left it there so its easy to find for when changes are made for a different ticket.